[clapack] clapack installs unusable config

Failed to build sample program with clapack.

**Environment**
Windows10, MS VS Community 2017 15.9.16.

**To Reproduce**
Install clapack: `vcpkg.exe install clapack:x64-windows`
Configure empty(just `main(){}`) sample program with CMakeLists.txt:
```
cmake_minimum_required(VERSION 3.12)
project(testclapack)
find_package(clapack CONFIG REQUIRED)
add_executable(testclapack test_clapack.cpp)
target_link_libraries(testclapack lapack)
```
Sample program build(Release x64) failed:
```
LINK : fatal error LNK1181: cannot open input file '\lib\openblas.lib'
```

**Cause**
Here the cut from `vcpkg/installed/x64-windows/share/clapack/clapack-targets.cmake`:
```
# Create imported target f2c
add_library(f2c STATIC IMPORTED)

# Create imported target lapack
add_library(lapack STATIC IMPORTED)

# Import target "f2c" for configuration "Release"
set_property(TARGET f2c APPEND PROPERTY IMPORTED_CONFIGURATIONS RELEASE)
set_target_properties(f2c PROPERTIES
  IMPORTED_LINK_INTERFACE_LANGUAGES_RELEASE "C"
  IMPORTED_LOCATION_RELEASE "D:/ws/vcpkg/probe/metisconfig/vcpkg/buildtrees/clapack/x64-windows-rel/F2CLIBS/libf2c/libf2c.lib"
  )

# Import target "lapack" for configuration "Release"
set_property(TARGET lapack APPEND PROPERTY IMPORTED_CONFIGURATIONS RELEASE)
set_target_properties(lapack PROPERTIES
  IMPORTED_LINK_INTERFACE_LANGUAGES_RELEASE "C"
  IMPORTED_LINK_INTERFACE_LIBRARIES_RELEASE "${_IMPORT_PREFIX}/lib/openblas.lib;f2c"
  IMPORTED_LOCATION_RELEASE "D:/ws/vcpkg/probe/metisconfig/vcpkg/buildtrees/clapack/x64-windows-rel/SRC/lapack.lib"
  )
```
Which is result of cmake `export(TARGETS ...)`, so it is relative to buildtree, and there is no `INTERFACE_INCLUDE_DIRECTORIES`, `IMPORTED_LOCATION_DEBUG`.

--- a/ports/clapack/CONTROL
+++ b/ports/clapack/CONTROL
@@ -1,5 +1,5 @@
 Source: clapack
-Version: 3.2.1-11
+Version: 3.2.1-12
 Homepage: https://www.netlib.org/clapack
 Description: CLAPACK (f2c'ed version of LAPACK)
 Build-Depends: openblas (!osx)

--- a/ports/clapack/fix-install.patch
+++ b/ports/clapack/fix-install.patch
@@ -1,0 +1,45 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 414ac8e..db58b4e 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -19,7 +19,7 @@ include_directories(${CLAPACK_SOURCE_DIR}/INCLUDE)
+ add_subdirectory(F2CLIBS)
+ add_subdirectory(SRC)
+ set(CLAPACK_VERSION 3.2.1)
+-export(TARGETS f2c lapack FILE ${CMAKE_INSTALL_PREFIX}/share/clapack/clapack-targets.cmake)
++install(EXPORT clapack-targets FILE clapack-targets.cmake DESTINATION share/clapack)
+ configure_file(${CLAPACK_SOURCE_DIR}/clapack-config-version.cmake.in
+ 	${CMAKE_INSTALL_PREFIX}/share/clapack/clapack-config-version.cmake @ONLY)
+ configure_file(${CLAPACK_SOURCE_DIR}/clapack-config.cmake.in
+diff --git a/F2CLIBS/libf2c/CMakeLists.txt b/F2CLIBS/libf2c/CMakeLists.txt
+index 6fa3598..c855e38 100644
+--- a/F2CLIBS/libf2c/CMakeLists.txt
++++ b/F2CLIBS/libf2c/CMakeLists.txt
+@@ -63,8 +63,9 @@ set_property(TARGET f2c PROPERTY PREFIX lib)
+ if(UNIX)
+   target_link_libraries(f2c m)
+ endif()
+-install(TARGETS f2c
++install(TARGETS f2c EXPORT clapack-targets
+         RUNTIME DESTINATION bin
+         LIBRARY DESTINATION lib
+-        ARCHIVE DESTINATION lib)
++        ARCHIVE DESTINATION lib
++        INCLUDES DESTINATION include)
+ 
+diff --git a/SRC/CMakeLists.txt b/SRC/CMakeLists.txt
+index 07dc8c7..2355225 100644
+--- a/SRC/CMakeLists.txt
++++ b/SRC/CMakeLists.txt
+@@ -377,8 +377,9 @@ if(BUILD_COMPLEX16)
+ endif()
+ add_library(lapack ${ALLOBJ} ${ALLXOBJ})
+ target_link_libraries(lapack ${BLAS_LIBRARIES} f2c)
+-install(TARGETS lapack
++install(TARGETS lapack EXPORT clapack-targets
+         RUNTIME DESTINATION bin
+         LIBRARY DESTINATION lib
+-        ARCHIVE DESTINATION lib)
++        ARCHIVE DESTINATION lib
++        INCLUDES DESTINATION include)
+ 

--- a/ports/clapack/portfile.cmake
+++ b/ports/clapack/portfile.cmake
@@ -14,6 +14,7 @@ vcpkg_extract_source_archive_ex(
   PATCHES
       remove_internal_blas.patch
 	  fix-ConfigFile.patch
+      fix-install.patch
 )
 
 vcpkg_configure_cmake(

